src/ucp/wireup/wireup.c

```diff
@@ -1465,10 +1466,10 @@ ucs_status_t ucp_wireup_send_pre_request(ucp_ep_h ep)
 
 ucs_status_t ucp_wireup_connect_remote(ucp_ep_h ep, ucp_lane_index_t lane)
 {
+    uct_ep_h uct_ep = ucp_ep_get_lane(ep, lane);
```

Comment:
1. move assert from line 1476 inside ucp_ep_get_lane()
2. if another function calling ucp_ep_get_lane() also asserts lane!=NULL_LANE (maybe in ucp_ep.inl getters) - such asserts could be removed as well

Reply:
2. There are several such places. I don't think we should remove the asserts in all of them.

Reply:
2. do you mean to have same assert both inside and outside ucp_ep_get_lane?

Reply:
No